### DISCUSSION TITLE
GitHub Pages only loads the index.html and nothing else

### DISCUSSION THREAD

**Discussion by djcaesar9114** [community/community#145976] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,
Here is my project: https://github.com/djcaesar9114/coursera_garden_party/tree/gh-pages
(there is a "main" branch but the github page is set with the "gh-pages" branch):

![image](https://github.com/user-attachments/assets/05198fc3-f130-4165-9e69-0c4c9570727b)

Unfortunately it doesn't work, because it doesn't seem to find all the assets:

![image](https://github.com/user-attachments/assets/7ca3add0-7244-4e0a-855e-a40f29fcd5fe)

I read the documentation and I did everything was explained to deploy correctly my React app.
Can anybody give me a hand here?
Thanks in advance.
César

**Comment by djcaesar9114** [comment]
I forced the basedir in the `index.html` file but I shouldn't have to do it.

**Comment by cachedcake** [comment]
Hi! @djcaesar9114 the solution should be the way you're directing the index.html to your assets folder, instead try this:

```html
<script type="module" crossorigin src="assets/index-9Bc6iXOR.js"></script>
<link rel="stylesheet" crossorigin href="assets/index-C9fRnurt.css">
``` 
The change I made was:
- Removing the hardcoded /coursera_garden_party/ prefix from asset path

Let me know if this helps!

**Reply by mecodeatlas** [reply]
Hey @djcaesar9114! Did @cakecable's response help you with your question? :)
